Tools/cases_generator/stacking.py

```diff
@@ -185,10 +186,12 @@ def __init__(
         self.final_offset = pred.final_offset.clone() if pred else StackOffset()
         for eff in reversed(instr.input_effects):
             self.final_offset.deeper(eff)
-            self.peeks.append(StackItem(offset=self.final_offset.clone(), effect=eff))
+            self.peeks.append(
+                StackItem(offset=self.final_offset.clone(), effect=eff))
```

Comment:
Lets reduce the diff
```suggestion
            self.peeks.append(StackItem(offset=self.final_offset.clone(), effect=eff))
```

Reply:
Ah I didn't realise but at some point during the buggy vscode fiasco it ran the autoformatter